src/board/servicies/board.service.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@nestjs/common';
+import { UserService } from 'src/user/servicies/user.service';
+import { BoardDao } from '../DAO/board.dao';
+import { CreateBoardDto } from '../dto/create-board.dto';
+import { UpdateBoardDto } from '../dto/update-board.dto';
+
+@Injectable()
+export class BoardService {
+  constructor(
+    private readonly boardDao: BoardDao,
+    private readonly userService: UserService,
+  ) {}
+
+  async findBoardById(id: string) {
+    return this.boardDao.getById(id);
+  }
+
+  async findAllBoards(auth: string) {
+    const user = await this.userService.findUserByToken(auth);
+    return this.boardDao.getAllUsersBoard(user._id.toHexString());
+  }
+
+  async createBoard(auth: string, dto: CreateBoardDto) {
+    const user = await this.userService.findUserByToken(auth);
+    const newBoard = await this.boardDao.createBoard({
+      ...dto,
+    });
+    newBoard.users = [user._id.toHexString()];
```

Comment:
move user._id.toHexString() into a varible (it is used more than one time - line 30)